python/.platform.app.yaml

```diff
@@ -0,0 +1,50 @@
+# This file describes an application. You can have multiple applications
+# in the same project.
+#
+# See https://docs.platform.sh/user_guide/reference/platform-app-yaml.html
+
+# The name of this app. Must be unique within a project.
+name: python
+
+# The runtime the application uses.
+type: "python:3.7"
+
+# The build-time dependencies of the app.
+dependencies:
+    python3:
+       pipenv: "2018.10.13"
+
+# The hooks executed at various points in the lifecycle of the application.
+hooks:
+    build: |
+      pipenv install --system --deploy
+
+# The size of the persistent disk of the application (in MB).
+disk: 1024
+# The relationships of the application with services or other applications.
+#
+# The left-hand side is the name of the relationship as it will be exposed
+# to the application in the PLATFORM_RELATIONSHIPS variable. The right-hand
+# side is in the form `<service name>:<endpoint name>`.
+relationships:
+    database: 'mysql:mysql'
+    postgresql: 'postgresql:postgresql'
+    solr: 'solr:solr'
+    redis: 'redis:redis'
+    elasticsearch: 'elasticsearch:elasticsearch'
+    mongodb: 'mongodb:mongodb'
+    rabbitmq: 'rabbitmq:rabbitmq'
+    influxdb: 'influxdb:influxdb'
+    memcached: 'memcached:memcached'
+
+# The configuration of app when it is exposed to the web.
+web:
+    commands:
+        start: python web/index.py
+
+    locations:
+        "/python":
+            # The public directory of the app, relative to its root.
+            root: "web"
+#            # The front-controller script to send non-static requests to.
+#            passthru: "python/index.py"
```

Comment:
I don't think a passthru is necessary on Python, is it?  I would expect this to be using uwsgi.  (I say that knowing almost nothing about Python best practices...)

Reply:
Not necessary, just a commented-out remnant. I'll remove.

Reply:
you need `passthru: true` for non-php apps